### PR TITLE
SGS flux bug fixes

### PR DESCRIPTION
**Pull request by yairchn**
remove extra a_env that was multiplying sgs diffusive flux and fix a bug in diagnostic mass flux of qt

### CODE REVIEW

**Review by charleskawczynski — APPROVED** [review]
Code changes look fine to me, would be good for others to verify the math

**Review by trontrytel — APPROVED** [review]

**Review by ilopezgp — APPROVED** [review]
LGTM

File: src/Turbulence_PrognosticTKE.jl [hunk 1]
```diff
@@ -101,7 +101,7 @@ function compute_sgs_tendencies!(
     # Adjust the values of the grid mean variables
     # Prepare the output
     @. massflux_tendency_h = -∇c(wvec(massflux_h)) * α0_c
-    @. massflux_tendency_qt = ∇c(wvec(massflux_qt))
+    @. massflux_tendency_qt = -∇c(wvec(massflux_qt)) * α0_c
     @. tends_θ_liq_ice += -∇c(wvec(massflux_h)) * α0_c
     @. tends_q_tot += -∇c(wvec(massflux_qt)) * α0_c
```
[the remark this answers is not in the document]
Comment (charleskawczynski) [reply src/Turbulence_PrognosticTKE.jl:106]: I think what's above is for diagnostics? They could (and probably be moved to diagnostics if we don't need them for runtime)

[the remark this answers is not in the document]
Comment (yairchn) [reply src/Turbulence_PrognosticTKE.jl:106]: let's leave it for a next PR?

**Comment by yairchn** [comment]
Bors r+

**Pull request merged**